packages/xrpl/src/models/methods/index.ts

```diff
@@ -276,12 +286,15 @@ type Response =
   // Price Oracle methods
   | GetAggregatePriceResponse
 
-export type RequestResponseMap<T> = T extends AccountChannelsRequest
+export type RequestResponseMap<
+  T,
+  Version extends APIVersion,
```

Comment:
@khancode is there a benefit to defaulting this value?

Reply:
`Version` is already being defaulted from its upper level in `Client.request()` so it's defaulted indirectly. The benefit of doing this is so TS can default to using API v2 response model. Otherwise the response model returned ends up being a union type `v1 | v2`.

Reply:
@khancode why not default it all the way down the stack?

Reply:
Sounds like a good design and brings added benefit. I just pushed the change.